openhexa/sdk/pipelines/runtime.py

```diff
@@ -50,7 +50,7 @@ def download_pipeline(url: str, token: str, run_id: str, target_dir: str):
               pipelineRun(id: $id) {
                 id
                 version {
-                  number
```

Comment:
Issue also solved in https://github.com/BLSQ/openhexa-sdk-python/pull/221 but not yet merged